src/integrationtests/resources/com/aws/greengrass/integrationtests/status/local_store_content/recipes/CustomerApp-1.0.0.yaml

```diff
@@ -7,6 +7,9 @@ ComponentVersion: '1.0.0'
 ComponentConfiguration:
   DefaultConfiguration:
     sampleText: This is a test
+ComponentDependencies:
```

Comment:
Is it required to add this? I understand removing it from below. But if we don't need this dependency for the tests to work, let's get rid of it.

Reply:
It helps testing things more thoroughly by deploying a component that's not root component